pandas/tests/series/indexing/test_setitem.py

```diff
@@ -135,6 +135,22 @@ def test_setitem_boolean_td64_values_cast_na(self, value):
         expected = Series([NaT, 1, 2], dtype="timedelta64[ns]")
         tm.assert_series_equal(series, expected)
 
+    def test_setitem_boolean_nullable_int_types(self, any_numeric_dtype):
+        # GH: 26468
+        ser = Series([5, 6, 7, 8], dtype=any_numeric_dtype)
+        ser[ser > 6] = Series(range(4), dtype=any_numeric_dtype)
+        expected = Series([5, 6, 2, 3], dtype=any_numeric_dtype)
+        tm.assert_series_equal(ser, expected)
+
+        ser = Series([5, 6, 7, 8], dtype=any_numeric_dtype)
+        ser.loc[ser > 6] = Series(range(4), dtype=any_numeric_dtype)
```

Comment:
is the failure on 3.9 something else? https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=47857&view=logs&j=f016abb9-7827-5fa2-935a-22bd9b1477b6&t=c86edbe1-8c1d-5e5c-5b6f-d970fa4acf6d

did you merge master recently?

cc @jbrockmendel

Reply:
Tuesday, so not really recently, merged now